app/views/courses/preview/_entry_requirements_qualifications.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-entry">Requirements</h2>
```

Comment:
why `entry` and not `requirements`

Reply:
Copied from C# as don't want to break any deep links

Reply:
can we add a tinny tiny tests for it please